convince-my-boss.txt

```diff
@@ -0,0 +1,15 @@
+﻿Dear {bossName},
+I’d like to attend the upcoming PyCon India 2018 conference taking place October 5-9, 2018 in Hyderabad, India. PyCon’s are premier conferences about developing and using the Python programming language. It attracts the best programmers, experts, enthusiasts, community leaders, entrepreneurs, and even college students from all corners of the country and abroad.
+The event focuses on key aspects of sharing knowledge and experience, solving important issues together, learning and collaborating, meeting and network with the best and brightest in the community in an open-source ecosystem.
+About a 1000+ of the industry's most dedicated and experienced individuals from different fields are bound to be in attendance for the 10th edition of this mega event. An opportunity like this is crucial for our company’s growth and development.
+Here's how my presence at PyCon India 2018 could effectively help us and our brand get exposed to an ocean full of opportunities that we are yet to explore.
+* PyCon India aims at bringing folks who love Python under one roof and create a healthy environment to learn. Fusing together fields of Data analysis, ML/AI, Automation, Web development, Software development and more.
+* It's a unique chance to create strategic partnerships and promote our company's brand at the forefront of the Python Community in India, by gathering valuable insights on current market trends and also recruiting fresh talent.
+* PyCon India is also a learning opportunity, offering a wide array of talks, workshops, keynotes, developer sprints strung together in a packed schedule each day divided by various tracks that people could follow.
+* Each presented by industry experts, community leaders and many young minds delving into new technologies and latest trends in the market. Some including ML/AI, Automation, Embedded Systems, Networking, Security, Web Development.
+* I will also be meeting top technical minds, brainstorm with them on ideas, challenges, and best practices. Collaborate with them on projects and solve technical issues.
+It is THE conference to go to for Python professionals. You can learn more about the conference and last year's talk and sessions here.

```

Comment:
Link in `here` missing. Use a link in brackets

Reply:
Good Catch.